### PR TITLE
Added missing is_stable argument to lax.sort

### PR DESCRIPTION
**Pull request by nova77**
While the [documentation](https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.sort.html#jax.lax.sort) states that `lax.sort` wraps XLA’s `Sort` operator, it was missing the `is_stable` argument. This PR fixes the inconsistency.

### CODE REVIEW

**Comment by jakevdp** [comment]
Looks great, thanks for the contribution! I'm going to run a more complete set of tests (including GPU/TPU) and then we can get this merged!

**Review by jakevdp — APPROVED** [review]

**Comment by jakevdp** [comment]
It looks like there are some subsequent changes that interact poorly with this. Could you rebase your branch on master?

**Comment by nova77** [comment]
> It looks like there are some subsequent changes that interact poorly with this. Could you rebase your branch on master?

Done (I synced master and merged it into this branch). Sorry about that, it's been a while since I used github with a forked repo.

File: tests/lax_test.py [hunk 1]
```diff
@@ -1338,54 +1338,58 @@ def testCumulativeReduce(self, op, onp_op, shape, dtype, axis, rng_factory):
...
         (jtu.device_under_test() == "cpu" and jax.lib.version <= (0, 1, 47)) or
          jtu.device_under_test() == "tpu")):
       raise SkipTest("Complex-valued sort not implemented")
     rng = jtu.rand_default(self.rng())
     args_maker = lambda: [rng(shape, dtype)]
-    op = lambda x: lax.sort(x, dimension=axis)
-    numpy_op = lambda x: lax_reference.sort(x, axis)
+    op = lambda x: lax.sort(x, dimension=axis, is_stable=is_stable)
+    def numpy_op(x):
+      return lax_reference.sort(x, axis, kind='stable' if is_stable else None)
```
Comment (jakevdp) [inline tests/lax_test.py:1377]: This is causing issues in numpy 1.16; the traceback is:
```
Traceback (most recent call last):
  File "<embedded stdlib>/unittest/case.py", line 59, in testPartExecutor
    yield
  File "<embedded stdlib>/unittest/case.py", line 605, in run
    testMethod()
  File "/build/work/269db3e22bc8b0a4438c43b03951abcd1c0e/google3/runfiles/google3/third_party/py/absl/testing/parameterized.py", line 282, in bound_param_test
    return test_method(self, **testcase_params)
  File "/build/work/269db3e22bc8b0a4438c43b03951abcd1c0e/google3/runfiles/google3/third_party/py/jax/tests/lax_test.py", line 1378, in testSortAgainstNumpy
    self._CheckAgainstNumpy(op, numpy_op, args_maker)
  File "/build/work/269db3e22bc8b0a4438c43b03951abcd1c0e/google3/runfiles/google3/third_party/py/jax/test_util.py", line 838, in _CheckAgainstNumpy
    lax_ans = lax_op(*args)
  File "/build/work/269db3e22bc8b0a4438c43b03951abcd1c0e/google3/runfiles/google3/third_party/py/jax/tests/lax_test.py", line 1377, in numpy_op
    return lax_reference.sort(x, axis, kind='stable' if is_stable else None)
  File "/build/work/269db3e22bc8b0a4438c43b03951abcd1c0e/google3/runfiles/google3/third_party/py/numpy/core/fromnumeric.py", line 934, in sort
    a.sort(axis=axis, kind=kind, order=order)
TypeError: expected bytes, NoneType found
```

Comment (nova77) [reply tests/lax_test.py]: Fixed. Sorry about that: is there a way to systematically test different versions of numpy besides creating an env for each one of them?

Comment (jakevdp) [reply tests/lax_test.py]: No, I don't know of any other shortcut for that.

**Comment by jakevdp** [comment]
Looks good, thanks!

**Pull request merged**